site2/website-next/blog/2022-12-01-pulsar-summit-asia-2022-recap.md

```diff
@@ -0,0 +1,108 @@
+
+* 1400+ registrations and 40,000+ views globally
+* 41 speakers from companies like ByteDance, Huawei, Tencent, Nippon Telegraph and Telephone Corporation (NTT), Yum China, Netease, vivo, Nutanix, and StreamNative
+* 3 keynotes on Apache Pulsar and event-driven applications
+* 36 sessions on use cases, technical deep dives, and ecosystem talks
+* 14 Program Committee members
+* 18 community and media partners
+
+
+## Keynotes and sessions at a glance
+
+This two-day virtual event brought together engineers, architects, and data scientists from the messaging and streaming communities. They talked about Pulsar adoption for different use cases, event-driven platforms, technical details, and even Pulsar integration with other ecosystems. The following is a quick recap of some of the keynotes and sessions.
+
+
+### Keynotes
+
+* **A Cloud-Native, Unified Messaging and Streaming System for Modern Data Infrastructure (Mandarin)**: Jia Zhai, an Apache Pulsar PMC member, gave a high-level overview of Apache Pulsar and explained how it meets the requirements for messaging and streaming with its cloud-native features.
+* **What You Should Know about Apache Pulsar in 2022 (Mandarin)**: Penghui Li, an Apache Pulsar PMC member, talked about some of the existing problems in Pulsar and how the Pulsar community would work to solve them going forward.
+* **Event-Driven Applications Done Right (English)**: Matteo Merli, Apache Pulsar PMC Chair, provided his insights on the fundamentals of modern event-driven applications.
+
+
+### Technical deep dives
+
+* **Taking Jakarta JMS to New Generation Messaging Systems - Apache Pulsar (English)**: Enrico Olivelli and Mary Grygleski from DataStax explained how Pulsar concepts map to the Jakarta Messaging Specifications and demonstrated how to connect a Jakarta EE application to Pulsar.
+* **A New Way of Managing Pulsar with Infrastructure as Code (Mandarin): **Max Xu and Fushu Wang from StreamNative discussed how to leverage the Terraform Provider for Pulsar and the Pulsar Resources Operator to help better manage Pulsar.
+* **A Deep Dive into Pulsar's Geo-replication for High Availability (Mandarin)**: Jialing Wang from China Mobile talked about the asynchronous and synchronous data replication mechanisms, and explained how they deployed Pulsar across multiple regions and improved its performance at China Mobile.
+* **Apache Pulsar in Volcano Engine E-MapReduce: Integration and Scenarios (Mandarin)**: Xin Liang from ByteDance introduced Volcano Engine E-MapReduce, a stateless, open-source big data platform, and how Pulsar fits into the platform’s ecosystem supporting different use cases.
+* **Handling 100K Consumers with One Topic: Practices and Technical Details (English): **Hongjie Zhai from NTT Software Innovation Center shared their practices and technical details of handling 100K consumers with a single Pulsar topic.
+
+
+### Use cases
```

Comment:
Shall we put `Use cases` ahead of `Technical deep dives` since it attracts more attention?

Reply:
Changed. Thanks.